SA/design_concept/content/design/functional_requirements.typ

```diff
@@ -37,38 +37,42 @@ By default, "user" in the use case description refers to the "student user".
     caption: "Use Case Diagram"
 )<use_case_diagram>
 
+As the aim of this project is to find a visual representation of functional programming,
+the use case descriptions are kept very brief and only state the intention behind the use case but not how it must be achieved.
```

Comment:
```suggestion
the use case descriptions are kept brief and only state the intention behind the use case, not how it must be achieved.
```

Not sure about the last sentence. I think a use case should never state how it is to be achieved

Reply:
Not in detail, but I think descriptions like 'click on a button', etc. are sometimes included. But I can also shorten the sentence.

Reply:
I took the to be achieved part out.